src/test/ssmDocument/explorer/registryItemNode.test.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as sinon from 'sinon'
+import { RegistryItemNode } from '../../../ssmDocument/explorer/registryItemNode'
+import { assertNodeListOnlyContainsErrorNode } from '../../utilities/explorerNodeAssertions'
+
+describe('RegistryItemNode', () => {
+    let sandbox: sinon.SinonSandbox
+
+    const fakeRegion = 'testRegion'
+    const names = ['Owned by Amazon', 'Owned by me', 'Shared with me']
+    const expectedChildNodeNames = ['Automation', 'Command']
+
+    beforeEach(() => {
+        sandbox = sinon.createSandbox()
+    })
+
+    afterEach(() => {
+        sandbox.restore()
+    })
+
+    it('initialized name and [View Only] correctly', async () => {
+        const testAmazonNode: RegistryItemNode = new RegistryItemNode(fakeRegion, names[0])
```

Comment:
This type of test is brittle, if the array were ever modified. Consider making consts for all array entries, and produce an array from those consts. That way your tests can directly use the consts or the array as needed.

The tests will read more intuitively too... "ConstFoo does this thing" vs "array index 4 does this thing"

Reply:
Makes sense. WIll change.